server/sonar-server/src/main/resources/org/sonar/server/component/ws/tree-example.json

```diff
@@ -5,13 +5,15 @@
     "total": 3
   },
   "baseComponent": {
+    "organization": "my-org-1",
     "id": "MY_PROJECT_ID",
     "key": "MY_PROJECT_KEY",
     "name": "Project Name",
     "qualifier": "TRK"
   },
   "components": [
     {
+      "organization": "my-org-1",
```

Comment:
Because "components" are in the same organization than "baseComponent", I think that the field "organization" should not be returned here.